src/main/org/codehaus/groovy/ast/expr/GStringExpression.java

```diff
@@ -32,13 +32,15 @@
  */
 public class GStringExpression extends Expression {
 
-    private String verbatimText;
-    private List<ConstantExpression> strings = new ArrayList<ConstantExpression>();
-    private List<Expression> values = new ArrayList<Expression>();
+    private final String verbatimText;
+    private final List<ConstantExpression> strings;
+    private final List<Expression> values;
     
     public GStringExpression(String verbatimText) {
         this.verbatimText = verbatimText;
         super.setType(ClassHelper.GSTRING_TYPE);
+        this.strings = new ArrayList<ConstantExpression>();
+        this.values = new ArrayList<Expression>();
```

Comment:
This would cause a breaking change if any code relied on `#getStrings` or `#getValues` returning `null`.  I would avoid this change.